### DISCUSSION TITLE
Locked Display Item

### DISCUSSION THREAD

**Discussion by AlmostPanda** [BG-Software-LLC/SuperiorSkyblock2#1835] [Pending Feature Requests]
Hi I would like to add new feature, which is Item in mission menu that will display "Locked Mission" with written lore "U have to complete mission ... to unlock this mission", this Item with Locked Mission display will appear before mission it self, and after completing certain mission this item will dissapear and will be replaced with Mission.
It might look something like this:
![Feature picture](https://github.com/BG-Software-LLC/SuperiorSkyblock2/assets/141587317/d737f492-4c7a-4f7b-a7a6-0df8ec9e073f)

**Comment by OmerBenGera** [comment]
Already a thing.

**Reply by AlmostPanda** [reply]
How can I do it?

**Reply by OmerBenGera** [reply]
You can edit the mission items inside the mission files.
